.github/workflows/data.yml

```diff
@@ -45,12 +45,12 @@ jobs:
       - id: cache_check
         uses: actions/cache@v3
         with:
-          path: ${{ needs.webbpsf_path.outputs.path }}
+          path: ${{ needs.path.outputs.path }}
```

Comment:
this was the offending line, for future reference (and other repos' workflows that may have the same problem)